server/server.go

```diff
@@ -16,6 +16,8 @@ import (
 
 // Version 3.6.0
 var v360 = semver.MustParse("3.6.0")
+var lastVersionForWindowsXP = "5.3.9"
+var lastVersionForOSXYosemite = "5.3.9"
```

Comment:
This can actually be 5.4.0 since I build that with the old go...

Reply:
Cool. Will change.

Reply:
Done